manta-accounting/src/wallet/mod.rs

```diff
@@ -171,6 +171,12 @@ where
         &self.ledger
     }
 
+    /// Returns a shared reference to the signer connection associated to `self`.
+    #[inline]
+    pub fn signer(&mut self) -> &mut S {
+        &mut self.signer
+    }
```

Comment:
Instead of doing this, you can just expose `self.signer` as `pub` in the struct definition.